parent/pom.xml

```diff
@@ -1097,6 +1097,7 @@
           <version>${plugin.version.proto-backwards-compatibility}</version>
           <configuration>
             <protoSourceRoot>${project.basedir}/src/main/proto</protoSourceRoot>
+            <allowBreakingChanges>true</allowBreakingChanges>
```

Comment:
same - can we add a comment in the clirr issue to remove this later (or make a new follow up issue)?